microsoft/testsuites/cpu/common.py

```diff
@@ -0,0 +1,131 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from __future__ import annotations
+
+from typing import Dict, List
+
+from lisa import BadEnvironmentStateException, Node
+from lisa.tools import Cat, Dmesg, Echo, Lscpu, Lsvmbus, Uname
+from lisa.util import SkippedException
+
+
+class CPUState:
+    OFFLINE: str = "0"
+    ONLINE: str = "1"
+
+
+def skip_test(node: Node) -> None:
+    uname = node.tools[Uname]
+    kernel_version = uname.get_linux_information().kernel_version
+    config_path = f"/boot/config-{kernel_version}"
+    config_result = node.execute(f"grep CONFIG_HOTPLUG_CPU=y {config_path}", shell=True)
+    if config_result.exit_code != 0:
+        raise SkippedException("This distro doesn't support cpu hot plug.")
+
+
+def set_vmbus_channels_target_cpu(node: Node) -> Dict[str, str]:
```

Comment:
please review all method names in this file, many of them are not straightforward. For example, in this method, it looks set all channels to cpu 0. This method name should specify it explicitly, and add comments to explain the purpose of this change.

Reply:
updated

Reply:
this method name is not updated.

Reply:
updated